src/js/app/modules/users/controllers/listUsers.controller.js

```diff
@@ -58,5 +58,29 @@
                 vm.postStatus.postedForm = true;
             });
         };
+
+        vm.getCertification = function (userIdentification, typeOfCertification){
+            console.log(userIdentification, typeOfCertification);
+            userIdentification = (userIdentification != undefined) ? userIdentification : '0';
```

Comment:
Why the value coercion to `0` when `userIdentification` is `undefined`? shouldn't you just return to stop right here?